src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,35 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class LoginPage extends BasePage {
+    //since we cannot inherit constructors in Java, we will need to call the super()
+    // (which refers to parent class)in our loginpage constructor
+    public LoginPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+    By emailField = By.cssSelector("input[type='email']");
+    By passwordField = By.cssSelector("input[type='password']");
+    By submitBtn = By.cssSelector("button[type='submit']");
+
+    public void
+    enterEmail() {//we used the generic findElement() method from basepage which has explicit wait
+        findElement(emailField).sendKeys("eduardo.santos@testpro.io");
+    }
+    public void enterPassword() {
+        findElement(passwordField).clear();
+        findElement(passwordField).sendKeys("te$t$tudent");
+    }
+    public void clickSubmit() {
+        click(submitBtn);
+    }
+
+
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._